mugoweb/ezpublish-legacy
# Starting legacy_2018 extension and moving enum datatype into it

The idea is to move code or features into an extension. Typically that is code that we're not interested in keeping but have to keep for backward compatibility.

For existing project you could just upgrade to the latest version and enable the new legacy extension. In case the legacy extension is only providing code and features you don't use in
the existing project, you could consider to not enable this extension.

Fo new projects, it would make sense to not enable the legacy extension.

The first thing to move into the extension is the datatype 'ezenum'.

==Testing instructions==
- Checkout branch
- change your settings/override/site.ini file to load the legacy_2018 extension
- rebuild autoloads
- clear cache
- go to the admin siteaccess
- confirm that you can add the enum datatype to a content class
- make sure the class edit/view templates are from the enum datatype are rendered
- make sure the content edit/view/diff templates are rendered for the enum datatype
- remove the attribute from the class
- disable the extension in settings/override/site.ini
- make sure that you cannot add an ezenum attribute to a content class
